src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -61,49 +61,16 @@ function isTimeZoneIntialized() {
 }
 
 /**
- * Format a number to be two digits
- */
-function formatDigit(num) {
-    return (num < 10 ? '0' : '') + num;
-}
-
-/**
- * Format a date object into DD/MM/YYYY format
- * @param date
- * @returns {String}
- */
-function convertDateToDDMMYYYY(date) {
-    return `${formatDigit(date.getDate())}/${formatDigit(date.getMonth() + 1)}/${date.getFullYear()}`;
-}
-
-/**
- * Format a date object into HHMM format
- * @param date
- * @returns {String}
- */
-function convertDateToHHMM(date) {
-    return formatDigit(date.getHours()) + formatDigit(date.getMinutes());
-}
-
-/**
- * To be run on page finish loading, this will select the input: start date,
- * start time, and timezone based on client's time.
+ * To be run on page finish loading, timezone based on client's time.
  *
  * The default values will not be set if the form was submitted previously and
  * failed validation.
  */
 function selectDefaultTimeOptions() {
     const now = new Date();
 
-    const currentDate = convertDateToDDMMYYYY(now);
-    const hours = convertDateToHHMM(now).substring(0, 2);
-    const currentTime = parseInt(hours, 10) + 1;
-    const timeZone = -now.getTimezoneOffset() / 60;
-
     if (!isTimeZoneIntialized()) {
-        $(`#${ParamsNames.FEEDBACK_SESSION_STARTDATE}`).val(currentDate);
-        $(`#${ParamsNames.FEEDBACK_SESSION_STARTTIME}`).val(currentTime);
```

Comment:
@whipermr5 I think we don't need to assign these default values because we are already assigning it while rendering new fsForm? 
In FeedbackSessionsForm.java
 `newFsForm.fsStartDate = feedbackSession == null
                              ? TimeHelper.formatDateForSessionsForm(TimeHelper.getNextHour())
                              : TimeHelper.formatDateForSessionsForm(feedbackSession.getStartTimeLocal());
`